collections/events.js

```diff
@@ -182,7 +182,7 @@ Events.updateGroups = function(eventId) {
 
 
 Meteor.methods({
-	saveEvent: function(eventId, changes, updateReplicas, sendNotifications) {
+	saveEvent: function(eventId, changes, updateReplicas, sendNotifications, addNotificationMessage) {
```

Comment:
the reason is, that the message from the organizer that will be added is just a part of the whole email message... so there are two understandings of "message". "organizerMessage" or "additionalMessage" is clearer.